package.json

```diff
@@ -26,8 +26,11 @@
     "iframe-resizer": "^3.5.14",
     "meetup-web-components": "3.3.1772",
     "raw-loader": "^0.5.1",
+    "react-copy-to-clipboard": "^5.0.1",
+    "swarm-constants": "1.2.58",
```

Comment:
last thing before merge; can we do `>1` as our constants version to make it easy to keep up to date with changes within the major version?